api/src/main/java/io/opentelemetry/trace/SpanId.java

```diff
@@ -107,6 +107,21 @@ public static SpanId fromBytes(byte[] src, int srcOffset) {
     return new SpanId(BigendianEncoding.longFromByteArray(src, srcOffset));
   }
 
+  /**
+   * Transforms a byte[] representation of a span id (eg. from a protobuf) and returns a base 16
+   * representation of the same.
+   *
+   * @param bytes The byte array containing the span id.
+   * @return A base 16 representation of the span id.
+   * @since 0.1.0
+   */
+  public static String asBase16(byte[] bytes) {
```

Comment:
What can I do to talk you into a nice `toHexString()` today?

Reply:
I'm going to make this match the rest of the methods in the class more closely, and rename it to `asLowerBase16`.